Invalid escape sequence in `intcircuits.py`

When I import from `schemdraw`, I get the following error:

```
    from schemdraw import Drawing, ImageFormat
TUB9piUY-py3.10/lib/python3.10/site-packages/schemdraw/__init__.py:1: in <module>
    from .schemdraw import Drawing, use, config, theme
TUB9piUY-py3.10/lib/python3.10/site-packages/schemdraw/schemdraw.py:11: in <module>
    from .elements import Element
TUB9piUY-py3.10/lib/python3.10/site-packages/schemdraw/elements/__init__.py:22: in <module>
    from .intcircuits import (IcPin, Ic, Multiplexer, IcDIP, VoltageRegulator, DFlipFlop, JKFlipFlop, Ic555,
E     File "TUB9piUY-py3.10/lib/python3.10/site-packages/schemdraw/elements/intcircuits.py", line 430
E       IcPin('$\overline{Q}$', side='right', slot='1/2', anchorname='Qbar')]
E             ^^^^^^^^^^^^^^^^
E   SyntaxError: invalid escape sequence '\o'
```

https://github.com/cdelker/schemdraw/blob/6c17a50e5a4454346de8a5f20907bff67c0996d9/schemdraw/elements/intcircuits.py#L430

Should this be a raw string (`r''`) instead?

List of valid escape sequences:

- https://docs.python.org/3/reference/lexical_analysis.html#escape-sequences

--- a/schemdraw/elements/intcircuits.py
+++ b/schemdraw/elements/intcircuits.py
@@ -427,7 +427,7 @@ class DFlipFlop(Ic):
         pins = [IcPin('D', side='left', slot='2/2'),
                 IcPin('>', side='left', slot='1/2'),
                 IcPin('Q', side='right', slot='2/2'),
-                IcPin('$\overline{Q}$', side='right', slot='1/2', anchorname='Qbar')]
+                IcPin(r'$\overline{Q}$', side='right', slot='1/2', anchorname='Qbar')]
 
         if preclr:
             pins.extend([IcPin('PRE', side='top', invert=preclrinvert),
@@ -458,7 +458,7 @@ class JKFlipFlop(Ic):
                 IcPin('>', side='left', slot='2/3'),
                 IcPin('K', side='left', slot='1/3'),
                 IcPin('Q', side='right', slot='3/3'),
-                IcPin('$\overline{Q}$', side='right', slot='1/3', anchorname='Qbar')]
+                IcPin(r'$\overline{Q}$', side='right', slot='1/3', anchorname='Qbar')]
 
         if preclr:
             pins.extend([IcPin('PRE', side='top', invert=preclrinvert),

